include/drivers/smmu/smmuv2.h

```diff
@@ -8,19 +8,23 @@
 
 #include <plat/machine/devices_gen.h>
 
+#if defined(CONFIG_PLAT_ZYNQMP)
+#define SMMU_PADDR           0xfd800000
+#elif defined(CONFIG_PLAT_TX2)
 /*the paddr address of the TX2 SMMU*/
-#define SMMU_TX2_PADDR                         0x12000000
+#define SMMU_PADDR                         0x12000000
```

Comment:
nitpick: could you please change the indent on this to align with `SMMU_PAGE_4KB` below (and the one for `CONFIG_PLAT_ZYNQMP`) as well